pages/algorithms/semantic-role.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Semantic Role 
+key: semantic-role
+---
+
+A semantic association that indicates an object's type. This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.
```

Comment:
I think:
> This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type. 
Should not be part of the definition but can be put into a note.

I think the definition should include something along the lines of "that can be determined by looking at the first valid value of the "role" attribute or by looking at the name of the html element if no role attribute is available.